simulator/README.md

```diff
@@ -83,10 +87,44 @@ Note:
 
 1. If you want to send a boolean in Event Details, please use non capital letters, for instance `true` or `false`.
 
+### Send Telemetry to TCP Client
+
+If you want to send telemetry to TCP Client, do:
+
+- Go to tab "Send Telemetry".
+- Write a name in Telemetry Id control.
+- Write details in Telemetry Details control.
+- Click "Send one instance Telemetry" to send the only one instance.
+- Read the telemetry in TCP Client.
+
+The structure of this telemetry is, for instance:
+
+| Telemetry Id |   Telemetry Details    |
+|:------------:|:----------------------:|
+|   elevation  | "compName" : "MTMount" |
+|              |                        |
+
+You can send another Telemetry Id with other Telemetry Details.
+
+- Click "Publish Telemetry" for telemetry published with a Frecuency (Hz) and a Duration (sec).
+
+Note:
+
+1. Before publish this telemetry you have to subscribe to this telemetry in TCP Client.
+2. It is recommended add more time in Duration option when subscribe to this telemetry in TCP Client.
```

Comment:
It is recommended add more time ... --> It is recommended to add more time ...

Reply:
Done. Thanks!